### DISCUSSION TITLE
get sensitive of a point in FFD box

### DISCUSSION THREAD

**Discussion by Zcaic** [mdolab/adflow#265] [General]
hello， if I have some points in FFD box，how can i get their sensitive according to sensitive of wing sensitive？

**Comment by marcomangano** [comment]
Hello @Zcaic ,
I am not sure about your specific needs, but you can take a look at [this section of the MACH-Aero tutorial](https://mdolab-mach-aero.readthedocs-hosted.com/en/latest/machAeroTutorials/opt_aero.html#optimization-callback-functions) to understand how to call the primal and adjoint solvers in ADflow. If you have added the geometry object correctly, the geometric sensitivities will be also included in the ADflow sensitivity output.
The specific function that runs the adjoint solver is [this one](https://mdolab-mach-aero.readthedocs-hosted.com/en/latest/machAeroTutorials/opt_aero.html#optimization-callback-functions), but again I recommend to check the full tutorial to understand how to structure your scripts.

**Reply by Zcaic** [reply]
Hello~, Thanks for answering me. for example, after I call primal and adjoint solvers in ADflow, I will get geometric sensitivities, then I have some additional points near the wing surface that have know the coordinates， I want to project geometric sensitivities to these points，so that I can observe the sensitivity of these points。
